modules/Settings/Proxy/models/Module.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Settings proxy config module model.
+ *
+ * @package   Settings.Module
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author Arkadiusz Sołek <a.solek@yetiforce.com>
+ */
+
+class Settings_Proxy_Module_Model extends Settings_Vtiger_Module_Model
+{
+	/**
+	 * Load proxy module list fields.
+	 */
+	public $listFields = [
+		'proxyConnection' => 'LBL_PROXY_CONNECTION',
+		'proxyProtocol' => 'LBL_PROXY_PROTOCOL',
+		'proxyHost' => 'LBL_PROXY_HOST',
+		'proxyPort' => 'LBL_PROXY_PORT',
+		'proxyLogin' => 'LBL_PROXY_LOGIN',
+		'proxyPassword' => 'LBL_PROXY_PASSWORD',
+	];
+
+	/**
+	 * Function to initiation.
+	 *
+	 * @throws \ReflectionException
+	 */
+	public function init()
+	{
+		foreach ($this->listFields as $fieldName => $fieldData) {
+			$value = \App\Config::security($fieldName);
+			$this->set($fieldName, $value);
+		}
+	}
+
+	/**
+	 * Function to get the instance of Config module model.
+	 *
+	 * @param mixed $name
+	 *
+	 * @throws \ReflectionException
+	 *
+	 * @return \Settings_Proxy_Module_Model|\Settings_Vtiger_Module_Model
```

Comment:
```suggestion
	 * @return self
```
or @return Settings_Proxy_Module_Model